sysutils/synkron/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           qmake5 1.0
+
+name                synkron
+version             1.6.2+r200
+revision            0
+
+categories          sysutils archivers
+platforms           darwin
+license             GPL-2+
+maintainers         nomaintainer
+
+description         Synkron is an application for synchronising folders
+long_description    Synkron is an application that helps you keep your files and folders always \
+                    updated. You can easily sync your documents, music or pictures to have their \
+                    latest versions everywhere. \
+                     \
+                    Synkron provides an easy-to-use interface and a lot of features. Moreover, it \
+                    is free and cross-platform.
```

Comment:
If you want that to look like a new paragraph in `port info`,  you need to add two newline characters to the beginning of the new paragraph, i.e. `\n\nSynkron provides an easy-to-...` Also, I'm not sure these two short sentences are really even necessary, since it applies to nearly every single piece of software that we make available on MacPorts. 😄

Reply:
changed, thanks for the tip.